keps/sig-scheduling/3633-matchlabelselectors-to-podaffinity/README.md

```diff
@@ -420,9 +475,12 @@ https://storage.googleapis.com/k8s-triage/index.html
 We expect no non-infra related flakes in the last month as a GA graduation criteria.
 -->
 
-- These e2e tests will be added.
-  - `MatchLabelKeys: ["pod-template-hash"]` in `PodAffinity` (both in Filter and Score) works as expected with a rolling upgrade scenario.
-  - `MatchLabelKeys: ["pod-template-hash"]` in `PodAntiAffinity` (both in Filter and Score) works as expected with a rolling upgrade scenario.
+N/A
```

Comment:
Not related to matchLabelSelectors, but we decided not to have e2e test in https://github.com/kubernetes/kubernetes/pull/116065.